yacron/job.py

```diff
@@ -235,15 +235,29 @@ async def report(
                 cmd = [shell_config["command"]]
 
         # pass the necessary information as env variables
+
+        # We have to be a bit careful because job.stderr and job.stdout
+        # can potentially be very large. On Linux there are limits
+        # both on the individual as well as combined length of the arguments.
+        std_err_str = job.stderr if job.stderr is not None else ""
+        std_out_str = job.stdout if job.stdout is not None else ""
+        # this is an arbitrary safe lower limit
+        max_length_arg = 1024 * 16
+        args_too_long = len(std_err_str) > max_length_arg or \
+                        len(std_out_str) > max_length_arg or \
+                        len(std_err_str) + len(std_out_str) > max_length_arg
+        std_err_str = std_err_str if not args_too_long else ""
```

Comment:
I wonder if, instead of truncating to an empty string, we could not truncate to some length, to allow us to pass at least part of the stdin/stdout?

I would then suggest to fallback to `std_err_str[:max_length_arg//2]` instead of `""` in case the limit is exceeded.  What do you think?

Reply:
Yeah I agree with the sentiment of making at least part of it available. I think we should add new flags then that signal whether all the content was passed or only part of it (e.g. YACRON_STDERR_TRUNCATED and YACRON_STDOUT_TRUNCATED). Because otherwise there is no way for the called shell script to determine what is happening.
Would that work for you?

Reply:
Cool I will implement that quickly; probably won't get to it until end of next week though when I have access to my machine again.
(Feel free to do it yourself if you want to have this pull request closed)